classes/PodsAPI.php

```diff
@@ -6279,7 +6316,29 @@ public function load_pods( $params = null ) {
 			}
 		}
 
-		return $this->load_object_pods( $params );
+		$loaded_pods = $this->load_object_pods( $params );
+
+		if ( empty( $loaded_pods ) ) {
+			return [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: e6a5fe812f09a0eb646addeb63eb1e8a</sub>